optima/results.py

```diff
@@ -1028,50 +1078,86 @@ def exporttoexcel(filename=None, outdict=None):
     workbook = Workbook(filename)
     
     for key,outstr in outdict.items():
-        worksheet = workbook.add_worksheet(sanitizefilename(key)) # A valid filename should also be a valid Excel key
+        if key == 'Optimal':
+            key = 'Optimized'  # Hack to replace optimal with optimized for sheet name
```

Comment:
Why is this being done? If we want to rename "Optimal" to "Optimized" gobally then we can, but I think we should do it consistently, otherwise it may cause problems and inconsistencies down the track.

Reply:
@sherriekelly requested this specifically, maybe @cliffckerr and @robynstuart can chat with @sherriekelly to reach a consensus?